netbox/circuits/api/serializers.py

```diff
@@ -44,13 +44,15 @@ class CircuitSerializer(TaggitSerializer, CustomFieldModelSerializer):
     status = ChoiceField(choices=CircuitStatusChoices, required=False)
     type = NestedCircuitTypeSerializer()
     tenant = NestedTenantSerializer(required=False, allow_null=True)
+    termination_a = NestedCircuitTerminationSerializer(read_only=True)
+    termination_z = NestedCircuitTerminationSerializer(read_only=True)
     tags = TagListSerializerField(required=False)
 
     class Meta:
         model = Circuit
         fields = [
             'id', 'cid', 'provider', 'type', 'status', 'tenant', 'install_date', 'commit_rate', 'description',
-            'comments', 'tags', 'custom_fields', 'created', 'last_updated',
+            'termination_a', 'termination_z', 'comments', 'tags', 'custom_fields', 'created', 'last_updated',
```

Comment:
The API view's queryset needs to be extended to prefetch these values. See `circuits.views.CircuitListView` for reference.

Reply:
Done. Thanks for catching this.